extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.property.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property class",
+  "type": "object",
+  "description": "A property class.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "description": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The description of the property. The string is UTF-8 encoded."
+      },
+      "elementType": {
+          "type": "string",
+          "enum": [
+              "SCALAR", "ARRAY", "VARIABLE_SIZE_ARRAY", "STRING", "BINARY"
+          ],
+          "description": "The type of each element of the property array. This value may be a `SCALAR` (boolean, integer, or float types), or it may be an array type. If `SCALAR`, `ARRAY`, or `VARIABLE_SIZE_ARRAY` is used, then `componentType` must also be specified. `ARRAY` is a fixed-length array, and requires a `componentsPerElement` property. `VARIABLE_SIZE_ARRAY`, `STRING`, and `BINARY` also require a corresponding `offsetsBufferView` to indicate where each element starts in the buffer. `STRING` is similar to a `VARIABLE_SIZE_ARRAY` of `UINT8`, except that the contents will be a UTF-8 encoded string. `BINARY` is an alias for a `VARIABLE_SIZE_ARRAY` of `UINT8`. This adds semantic meaning that the element is used as a binary blob, rather than an array of numbers",
+          "default": "SCALAR"
```

Comment:
For clarity and consistency across all property declarations, I think we should remove the default `elementType`. Definitely a nitpick, but it's something I noticed as I was writing examples for `3DTILES_metadata`.

Reply:
@lilleyse and I talked about this in a call this morning, we like this idea. a little more verbose but it's more explicit.